packages/openactive-integration-tests/test/features/details-capture/customer-details-capture-non-essential/README.md

```diff
@@ -7,9 +7,28 @@ Support for capturing forename, surname, and telephone number from the Customer.
 https://www.openactive.io/open-booking-api/EditorsDraft/#customer-details-capture
 
 Coverage Status: **none**
+### Test prerequisites
+Opportunities that match the following criteria must exist in the booking system (for each configured `bookableOpportunityTypesInScope`) for the configured primary Seller in order to use `useRandomOpportunities: true`. Alternatively the following `testOpportunityCriteria` values must be supported by the [test interface](https://openactive.io/test-interface/) of the booking system for `useRandomOpportunities: false`.
 
+[TestOpportunityBookablePaid](https://openactive.io/test-interface#TestOpportunityBookablePaid) x3, [TestOpportunityBookable](https://openactive.io/test-interface#TestOpportunityBookable) x1
 
 *Note the test coverage for this feature is currently nonexistent. The test suite does not yet include non-stubbed tests for this feature.*
 
 
+## 'Implemented' tests
+
+Update `default.json` within `packages/openactive-integration-tests/config/` as follows to enable 'Implemented' testing for this feature:
+
+```json
+"implementedFeatures": {
+  ...
+  "customer-details-capture-non-essential": true,
```

Comment:
We should also update this in `default.json` as we introduce each new test, otherwise they won't run in CI

`"customer-details-capture-non-essential": true`

Reply:
Done, that default also has changes for seller urls, that is why it was overlooked, but I will pay attention in new tests :)

Reply:
Perfect!